components/omega/src/infra/logging.h

```diff
@@ -0,0 +1,108 @@
+// Omega logging header file.
```

Comment:
Similarly, start logging with capital L, use standard llvm-style header. In the header files, we also want doxygen style comments to document this file, so add a file description at least and document the interfaces below if possible (not sure how doxygen works with MACROs)

Reply:
OK